src/string_bytes.cc

```diff
@@ -150,62 +150,83 @@ static const int unbase64_table[] =
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
   };
-#define unbase64(x) unbase64_table[(uint8_t)(x)]
+#define unbase64(x)                                                           \
+    static_cast<uint8_t>(unbase64_table[static_cast<uint8_t>(x)])
```

Comment:
Yes.  The alternative was to change the type of the list elements to `uint8_t` but then I'd also have to change all the `-1` to `255` to squelch compiler warnings.  It would make the diff a lot noisier for no reason; conversion from signed to unsigned is well-defined.